src/app/app.component.html

```diff
@@ -1,5 +1,8 @@
 <app-header></app-header>
 <div class="wrapper">
   <router-outlet></router-outlet>
+  <div class="content-wrapper" style="padding-top: 200px">
+    <app-slider></app-slider>
+  </div>
```

Comment:
When user will navigate our app, this block will be present on each page, we don't want it. We want this block to be visible only on the main page - "airways.com/", so we need to refactor the code.
Add empty not-lazy route in `app-routing-module` and assign to this route `main` component, where we will place all components from the main page.

_app-routing-module_
```
{
 path: '',
 component: MainComponent
}
```

_main.component.html_
```
<div class="content-wrapper" style="padding-top: 200px">
 <app-slider></app-slider>
 <app-second-block>
 <app-third-block>
 ...
</div>
```

Reply:
It's just for viewing how it works. It's not an independent component and we delete it from app.component.

Reply:
I have deleted slider from app.component